main.ts

```diff
@@ -218,12 +243,29 @@ function renderAcademics(options) {
 }
 
 function renderBackground(options) {
-  renderHorizontalBarChat(d3.select('#ethnicity'), ETHNICITY, 400, 300, true);
+  renderMultiSeriesHorizontalBarChat(d3.select('#ethnicity'), ETHNICITY, 400, 300, true, {"ethnicity-all": 0, "ethnicity-women": 1, "ethnicity-men": 2});
   renderPieChart(d3.select('#international'), INTERNATIONAL, options.width * 0.75, options.width * 0.75);
   renderHorizontalBarChat(d3.select('#parent-education'), PARENT_EDUCATION, options.width, 280, true);
   renderPieChart(d3.select('#gender'), GENDER, options.width * 0.75, options.width * 0.75);
-  renderHorizontalBarChat(d3.select('#family-income'), FAMILY_INCOME, 400, 200);
-  drawWordCloud(d3.select('#hs-extras'), HS_EXTRAS, options);
+  renderHorizontalBarChat(d3.select('#family-income'), FAMILY_INCOME, 400, 300, false);
+  renderPieChart(d3.select('#year-of-birth'), YEAR_OF_BIRTH, options.width * 0.75, options.width * 0.75);
+  renderPieChart(d3.select('#sexual-orientation'), SEXUAL_ORIENTATION, options.width * 0.75, options.width * 0.75);
+  renderHorizontalBarChat(d3.select('#home-location'), HOME_LOCATION, 400, 300, false);
+  renderPieChart(d3.select('#immigrated'), IMMIGRATED, options.width * 0.75, options.width * 0.75);
+  renderHorizontalBarChat(d3.select('#siblings'), SIBLINGS, 400, 300, false);
+  renderHorizontalBarChat(d3.select('#enriched-program'), ENRICHED_PROGRAM, 400, 300, false);
+  renderPieChart(d3.select('#cegep'), CEGEP, options.width * 0.75, options.width * 0.75);
+  renderHorizontalBarChat(d3.select('#cegep-attended'), CEGEP_ATTENDED, 400, 300, true);
+  renderHorizontalBarChat(d3.select('#mother-tongue'), MOTHER_TONGUE, 400, 300, true);
+  renderPieChart(d3.select('#programming'), PROGRAMMING, options.width * 0.75, options.width * 0.75);
+  renderPieChart(d3.select('#cat-or-dog'), CAT_OR_DOG, options.width * 0.75, options.width * 0.75);
+  renderDotPlot(d3.select('#admission-average'), ADMISSION_AVERAGE, options.width, 400, {
```

Comment:
Would a histogram or bar graph suit this better?  We should leave a TODO to try out the histogram once #3 is merged.

Reply:
It would. Will do that once #3 merges